src/pages/admin/ui/colors/color-presentation/ColorPresentation.vue

```diff
@@ -70,15 +71,16 @@
   const hiddenInput = ref()
 
   function colorCopy() {
-    hiddenInput.value.select()
-    document.execCommand('copy')
-    notify()
+    navigator.clipboard?.writeText?.(hiddenInput.value.value).then(notify)
   }
 
+  const { init } = useToast()
```

Comment:
Good thing you noticed that. Otherwise would've probably been a runtime error :+1: